src/_locales/en_US/messages.json

```diff
@@ -369,6 +369,10 @@
         "message": "Send websites the \"<a href='https://www.eff.org/issues/do-not-track' target='_blank'>Do Not Track</a>\" signal",
         "description": "Checkbox label for enabling/disabling the Do Not Track signal, found on the general settings page"
     },
+    "options_alternateErrorPagesEnabled_checkbox": {
+        "message": "Re-enable the alternateErrorPagesEnabled Chrome Browser setting",
```

Comment:
How does this read to a non-technical Privacy Badger user? "What the heck is alternateErrorPagesEnabled??"

We should look up the language Chrome itself used by finding this setting under advanced privacy settings in an older Chrome version (before Chrome removed this toggle from privacy settings), or perhaps in [Chrome code history](https://source.chromium.org/chromium/chromium/src).

Reply:
I took the wording they use in [extensions privacy docs](https://developer.chrome.com/extensions/privacy) and rephrased it a bit to try and get the point across that a third party is implied. What do you think of: "Disable Chrome's 'alternateErrorPagesEnabled' that help resolve navigation errors by sharing your navigation data with a third party service" ?